LOCAL_VIRTUALENV.rst

```diff
@@ -122,21 +141,6 @@ Simply enter the Breeze environment by using ``workon`` and, once you are in it,
 
   ./breeze --initialize-local-virtualenv
 
-Optional - create a local virtualenv with conda
------------------------------------------------
-
-- install `miniconda3 <https://docs.conda.io/en/latest/miniconda.html>`_
-
-.. code-block:: bash
-
-  conda create -n airflow python=3.6
-  conda activate airflow
-  pip install -U -e ".[devel,gcp,postgres,ssh]"
-
-  # if necessary, start with a clean AIRFLOW_HOME, e.g.
-  # rm -rf ~/airflow
-  airflow db init
```

Comment:
Why you skipped db init?

Reply:
This is repeated in the original doc for the conda section. Under the change of this PR, I grouped conda and virtualenvwrapper as two different ways to create virtual environment. So the suggested change will follow something like this.

1. Create local virtualenv 
    - using virtualenv wrapper
    - using conda
2. Pip install
3. airflow db init